site/content/en/docs/Integration Patterns/player-capacity.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Allocating based on GameServer Player Capacity"
+linkTitle: "Player Capacity"
+date: 2021-08-31
+weight: 90
+description: >
+  Find a `GameServer` that has room for a specific number of players.
+---
+
+{{< alpha title="Player Tracking, Allocation Player Filter and Allocation State Filter" gate="PlayerTracking,PlayerAllocationFilter,StateAllocationFilter" >}}
```

Comment:
nit: I'd use an oxford comma in the title here: "Player Tracking, Allocation Player Filter, and Allocation State Filter".